src/sentry/web/frontend/auth_login.py

```diff
@@ -82,11 +209,32 @@ def get_auth_provider(self, organization_slug):
 
         return auth_provider
 
-    def get_login_form(self, request: Request):
-        op = request.POST.get("op")
-        return AuthenticationForm(request, request.POST if op == "login" else None)
+    def handle_register_form_submit(
+        self, request: Request, context: dict, organization: Organization, **kwargs
+    ) -> HttpResponse:
+        """Validates a completed register form, redirecting to the next
+        step or returning the form with its errors displayed."""
+
+        register_form = self.get_register_form(
```

Comment:
I like `get_` since it returns the form, "initialize" makes me think it's starting a form, but actually the form is already complete and it's just fetching it out of the request. We could name it "fetch" but that term has other connotations re. Javascripts Fetch and like fetching from a DB. I think `get` is a safe bet but I agree it _could_ be better, just not sure what would be better

Reply:
🤔 

That's not quite the case though.
This is constructing the form instance from the request - so in this case i _would_ say that we are initializing a form instance with data from the request.

Remember the request is just a normal xhr request. It doesn't contain any forms, it doesn't contain anything special to django at all. Just data that we're abstracting out to construct these new representations of that data.

Reply:
ok im convinced. fixed!